auth/oauth.go

```diff
@@ -58,28 +73,24 @@ func (oa *OAuthFlow) ObtainAccessToken() (accessToken string, err error) {
 	defer resp.Body.Close()
 
 	var values url.Values
-	bb, err := ioutil.ReadAll(resp.Body)
-	if err != nil {
-		return
-	}
-	if resp.StatusCode == http.StatusOK || strings.HasPrefix(resp.Header.Get("Content-Type"), "application/x-www-form-urlencoded") {
+	if strings.Contains(resp.Header.Get("Content-Type"), "application/x-www-form-urlencoded") {
```

Comment:
Just wanted to double check, is this is the only valid `Content-Type` that could be returned here?